proxy/src/schema/registry.rs

```diff
@@ -1,82 +1,149 @@
-use futures::future::Future;
-
 use radicle_registry_client::{
-    ed25519::Pair, ClientT, CreateCheckpointParams, RegisterProjectParams, String32, H256,
+    self as registry, ed25519, message, Client, ClientT, CryptoPair, String32, TransactionExtra,
+    H256,
 };
 
 use crate::schema::error::{Error, ProjectValidation};
 
+/// A container to dissiminate and apply operations on the [`Registry`].
+#[derive(GraphQLObject)]
+pub struct Transaction {
+    /// Unique identifier, in actuality the Hash of the transaction. This handle should be used by
+    /// the API consumer to query state changes of a transaction.
+    pub id: juniper::ID,
```

Comment:
It seems to me that if `id` is the `hash`, the types should reflect that. or do you want, intentionally, want to keep that separation?

Reply:
We can use none juniper (the crate we use to drive our GraphQL API) supported types, but that needs custom `Scalar` implementations. I captured that in the follow-up issue #99.